### DISCUSSION TITLE
Data table with 3 frozen/sticky columns

### DISCUSSION THREAD

**Discussion by daveKoala** [vuetifyjs/vuetify#13052] [Q&A]
Has anyone managed to use data tables and have the first 1, 2 or 3 columns frozen in place and the remaining columns scrollable left/right and keep the headers on show?

I've come across  CSS examples and additional packages put i need to keep the functionality and styling the same as the current Vuetify data table.

Many thanks

**Comment by megaroeny** [comment]
This is already in a long discussion, and coming to v3: https://github.com/vuetifyjs/vuetify/issues/4652
